docs/production.md

```diff
@@ -85,13 +85,14 @@ After installing a Katello server, you could then spin up a Capsule with the ass
 ### Local Deployment
 
 1. ssh to target machine **as root**
-2. Install Ansible -- `yum install ansible`
+2. Install Ansible -- `yum install ansible` (this will require the EPEL repo)
```

Comment:
There are many places to get Ansible, (e.g. http://mirror.centos.org/centos/7/configmanagement/x86_64/ or RHEL repos) so perhaps we should leave this up to the user to decide.

Reply:
```suggestion
2. Install Ansible -- `yum install ansible` (for example from the EPEL repo)
```

Reply:
oh, I didn't mean to resolve the whole thing, just the "for example". I guess I will Unresolve it until I can verify that "Commit suggestion" option actually did what I wanted.

Reply:
...and now I see how the blocks are separated, so all g on this one, I think.